### ISSUE TITLE
Add local language to cheatsheet translations

### ISSUE THREAD

**Issue by crichID** [github/training-kit#544]
## Overview
**TL;DR**
If I come to the website looking for a cheatsheet in French, I should see Francais instead of French in the list of languages.

#### Screenshots

<img width="852" alt="screen shot 2017-08-10 at 8 52 13 am" src="https://user-images.githubusercontent.com/9950121/29171086-41f89a8e-7da9-11e7-8b0d-98dc46b6a159.png">

#### Reasoning
On the [cheat sheet page](https://services.github.com/on-demand/resources/cheatsheets/) we link to all of the translated cheat sheets using the English translations of the language. Per [a suggestion from a contributor](https://github.com/github/training-kit/pull/476#issuecomment-283583356), it would be a great idea to list these in the local language.

The English is really only there for us, so we could potentially move the english translation to an Alt tag and just keep the local language.

### How to Contribute
- [ ] Comment on this issue if you'd like to work/collaborate on it! 
- [ ] Fork the [repository](https://github.com/github/training-kit).
- [ ] Use the [GitHub Flow](https://guides.github.com/introduction/flow/) to make changes to your fork. [This](https://services.github.com/on-demand/intro-to-github/) is a refresher course if you're unsure about how to make a change on GitHub.
- [ ] Push your changes to your repository. 
- [ ] Submit a Pull Request
  - Base Dropdown: github/training-kit
  - Compare Dropdown: Your fork

#### Testing Locally
If you'd like to make and test changes locally (and see how they would look if merged), do the following:
- In your command line:
  - [Install Git](https://git-scm.com/)
  - [Install Ruby](https://www.ruby-lang.org/en/documentation/installation/)
  - [ ] Run `script/setup`
  - [ ] Run `script/server`.
    - When successful, the script will initiate a local server at `http://127.0.0.1:4000/on-demand`.

### Questions?
- Leave a comment on this issue! Make sure to use @ mentions if you want a specific person's attention!
